lib/services/irc.rb

```diff
@@ -137,6 +137,14 @@ def irc_puts(command, debug_command=command)
     writable_irc.puts command
   end
 
+  def irc_realname
+    repo_owner = payload["repository"]["owner"]["name"]
+    repo_name = payload["repository"]["name"]
+    repo_private = payload["repository"]["private"]
+
+    "GitHub IRCBot - #{repo_owner}" + (repo_private ? "" : "/#{repo_name}")
```

Comment:
Total nit, but my only suggestion here is to be a bit paranoid and explicitly check that `repo_private == false`. I don't think it would happen, but if `payload["repository"]["private"]` was ever `nil` for some reason we would default to disclosing the repo name. It would be best to default to not showing the repo name unless we are 100% sure that it is not private.